include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10707,6 +10707,15 @@
           "capabilities" : [ "FPGAKernelAttributesINTEL" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "RegisterMapInterfaceINTEL",
+          "value" : 6160,
+          "parameters" : [
+            { "kind" : "LiteralInteger", "name" : "'AcceptDownstreamStall'" }
+          ],
+          "capabilities" : [ "FPGAKernelAttributesINTELv2" ],
```

Comment:
We also need to add the new `FPGAKernelAttributesINTELv2` capability to the grammar file.

Reply:
I think the name of the capability should also be `FPGAKernelAttributesv2INTEL`?  In other words, the `INTEL` suffix should remain at the end of the capability name, and `v2` should be before the suffix.

Reply:
Added capability to the grammar file and renamed it here as well as in the extension doc. Had to open a new PR: https://github.com/KhronosGroup/SPIRV-Headers/pull/317 (because this was automatically closed by git when I tried syncing my fork)